tests/Tests/Polygon2d.elm

```diff
@@ -10,7 +11,8 @@ import Fuzz
 import Geometry.Expect as Expect
 import Geometry.Fuzz as Fuzz
 import LineSegment2d
-import Polygon2d
+import Point2d
+import Polygon2d exposing (convexHull)
```

Comment:
`exposing (convexHull)` isn't actually necessary here, right? (And arguably confusing since `convexHull` is used as a variable name later...)

Reply:
Ah that’s probably my overzealous IDE.